app/views/admin/organisations/_form.html.erb

```diff
@@ -76,18 +76,6 @@
       <%= organisation_form.label :parent_organisation_ids, 'Sponsoring organisations' %>
       <%= organisation_form.select :parent_organisation_ids, options_from_collection_for_select(Organisation.with_translations(:en) - [organisation_form.object], 'id', 'select_name', organisation.parent_organisation_ids), {}, multiple: true, class: 'chzn-select form-control', data: { placeholder: "Choose parent organisations…" } %>
     </p>
-
-    <h3>Policy Areas</h3>
-    <% organisation_form.object.organisation_classifications.each do |ot| %>
```

Comment:
This filter list is a weird mix of Topical events and Policy Areas so work will need to be done after Policy area's are retired to change the wording of the lists to 'Topical events'. And also test what happens when a policy area that an org is tagged to is retired.